src/app/player-creation/player-creation.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
```

Comment:
Can you put this under a folder called services, please?

Reply:
yes mam